### PR TITLE
Network Demos

### PR DESCRIPTION
**Pull request by willtome**
This category of demos shows examples of network operations and management with Ansible Automation Platform.

This PR adds the network resource [repo](https://github.com/nleiva/ansible-net-modules) by @nleiva that uses Cisco DevNet Always on IOS, IOSXR, NXOS as endpoints for deploying golden configs by resource.

### CODE REVIEW

File: network/setup.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    source_project: Network Golden Configs
+    source_path: hosts
+
+controller_templates:
+  - name: NETWORK / Configuration
+    organization: Networking
+    inventory: Network Inventory
+    survey_enabled: true
+    project: Network Golden Configs
+    playbook: main.yml
```
Comment (MKletz) [inline network/setup.yml:44]: Is main.yml coming in a different PR?

Comment (termlen0) [reply network/setup.yml:44]: Mike, I see the main.yml here: https://github.com/nleiva/ansible-net-modules/blob/main/main.yml in Nick's repo

**Comment by termlen0** [comment]
Tested the PR and validated it working correctly

**Review by termlen0 — APPROVED** [review]
Tested and validated

**Pull request merged**